src/app/components/AuthInitializer/index.tsx

```
@@ -5,17 +5,17 @@ import { useEffect } from "react";
 };
 
 export default function AuthInitializer({ isLoggedIn }: Props) {
-  const setIsLoginUser = useSetAtom(isLoginUserAtom);
+	const setIsLoginUser = useSetAtom(isLoginUserAtom);
 
-  useEffect(() => {
-    if (isLoggedIn !== null) {
-      setIsLoginUser(isLoggedIn);
-    }
-  }, [isLoggedIn]);
+	useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: setIsLoginUser

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)